File: src/kyu_7/leap_years/csharp/leap_years_test.cs
using NUnit.Framework;

[TestFixture]
public static class LeapYear
{
  [Test]
  public static void TestYear2020()
  {
    Assert.AreEqual(true, Kata.IsLeapYear(2020));
  }
  
  [Test]
  public static void TestYear2000()
  {
    Assert.AreEqual(true, Kata.IsLeapYear(2000));
  }
  
  [Test]
  public static void TestYear2015()
  {
    Assert.AreEqual(false, Kata.IsLeapYear(2015));
  }
  
  
  [Test]
  public static void TestYear2100()
  {
    Assert.AreEqual(false, Kata.IsLeapYear(2100));
  }
}

File: src/kyu_7/leap_years/python/leap_years_test.py
from solution import is_leap_year
import codewars_test as test

@test.describe("Example tests")
def example_tests():

    @test.it("Year 2020 is a leap year")
    def test_year_2020():
        test.assert_equals(is_leap_year(2020), True, "Incorrect answer for year = 2020")

    @test.it("Year 2000 is a leap year")
    def test_year_2000():
        test.assert_equals(is_leap_year(2000), True, "Incorrect answer for year = 2000")

    @test.it("Year 2015 is not a leap year")
    def test_year_2015():
        test.assert_equals(is_leap_year(2015), False, "Incorrect answer for year = 2015")

    @test.it("Year 2100 is not a leap year")
    def test_year_2100():
        test.assert_equals(is_leap_year(2100), False, "Incorrect answer for year = 2100")
